### ISSUE TITLE
"Mark as done" check overlapping Archive icon

### ISSUE THREAD

**Issue by megaroeny** [leggett/simplify#822]
Recently, I noticed that the ✔️ overlaps the 🗄️ Archive icon, when the "Mark as done" option is enabled:

<img width="94" alt="image" src="https://github.com/user-attachments/assets/5acf185d-461e-48ce-af3a-084163245fd4">

- This does not happen on my Windows - Edge browser.
- I'm on v3.1.14 using Arc browser on macOS 14.5. 

---

Here's my debug log I copied if that helps:
```
Simplify v.3.1.14 configuration: aAX avatarsInList avatarsZoom boldHighlight bundleInbox caA caI caL cfC cfF cfO cfR cfU composeActions composeFab debug defaultTheme fontList fontMsg fontNav geminiMin genSumMin hideAds hideEmptySections hideExtWarn hideFileIcons hideInboxHideAllLink hideLabelChips hideListCount hideMsgCount hideMsgReplies hideTabIcons importantBadge inInbox inList invertAddons invertCompose izBg2 lightTheme listWidthLg lowDensity maDone maPin matYou msgWidthSm nPane newNav newUI oeGemini otherExtensions reminders rightImportance rightLabels rightStar sendLater simplify simplifyCanary simplifyPrefsOpen unreadDot- 

System: Mozilla/5.0 (Macintosh Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 - gmail.pinto-server_20240715.06_p0 - Window: 1078 x 892 (131pct zoom) - Language: en - Errors: / 2232:49: doWhenReady(.aiL .a7J) timed out / 2416:22: Something went wrong with the click series / 0:0: Something went wrong when trying to dynamically change the theme / 2221:54: clickOnWhenReady(.QW .OD, div[aria-label="Theme"] + button) timed out / 3128:14: SGP.CH failed / 12471:11: Could not find current section based on focusedItem
```

**Comment by leggett** [comment]
I'm not able to reproduce this (using Arc on a Mac with v3.1.14). I wonder if a hard-refresh (shift+command+R) fixes it?

**Comment by megaroeny** [comment]
> I'm not able to reproduce this (using Arc on a Mac with v3.1.14). I wonder if a hard-refresh (shift+command+R) fixes it?

It didn't unfortunately. I cleared cookies and logged out as well. I also updated to `v.3.1.15`, still there. ☹️ 

If it helps, I'm on the Early Bird version of Arc: Version `1.51.0 (51691)-RC`

**Comment by leggett** [comment]
I think I found the issue -- Gmail is rolling out a change that I don't have on my main accounts. I found it on one of my test accounts and will work on a fix ASAP. Thanks for the report.

**Comment by megaroeny** [comment]
> I think I found the issue -- Gmail is rolling out a change that I don't have on my main accounts. I found it on one of my test accounts and will work on a fix ASAP. Thanks for the report.

Sweet. No worries, my pleasure to contribute Michael.

**Comment by megaroeny** [comment]
Confirmed this is happening in Edge on Windows too, so not just an Arc/macOS issue.

**Comment by leggett** [comment]
Fix is live in v3.1.16 on Canary and Firefox. I expect Chrome (non-canary) and Safari to be live within 24 hours and Edge in a couple weeks (Microsoft takes FOREVER with their reviews).

**Comment by megaroeny** [comment]
> Fix is live in v3.1.16 on Canary and Firefox. I expect Chrome (non-canary) and Safari to be live within 24 hours and Edge in a couple weeks (Microsoft takes FOREVER with their reviews).

Thanks Michael!
